src/hapi_schema/db_national_risk.py

```diff
@@ -9,15 +9,12 @@
     Float,
     ForeignKey,
     Integer,
+    String,
     select,
 )
 from sqlalchemy.orm import Mapped, mapped_column, relationship
 
-from hapi_schema.db_admin1 import DBAdmin1
-from hapi_schema.db_admin2 import DBAdmin2
-from hapi_schema.db_dataset import DBDataset
 from hapi_schema.db_location import DBLocation
-from hapi_schema.db_resource import DBResource
```

Comment:
These have already been removed in develop, something strange with the merge maybe, it will probably be fine

Reply:
Not sure about this one -